site/_docs/configuration.md

```diff
@@ -339,6 +349,34 @@ before your site is served.
   </p>
 </div>
 
+## Specifying a Jekyll environment at build time
+
+In the build (or serve) arguments, you can specify a Jekyll environment and value. The build will then apply this value in any conditional statements in your content. 
+
+For example, suppose you set this conditional statement in your code:
+
+```liquid
+ {% raw %}
+ {% if jekyll.environment == "production" %}
+ {% include disqus.html %} 
+ {% endif %}
```

Comment:
Would you mind indenting so that `{% if...` is 2 spaces in and `{% include` is 4 spaces in?